tests/acceptance/features/coreApiShareOperationsToShares1/gettingSharesPendingFiltered.feature

```diff
@@ -18,20 +18,42 @@ Feature: get the pending shares filtered by type (user, group etc)
     And user "Alice" has uploaded file with content "file to share with user" to "/fileToShareWithUser.txt"
     And user "Alice" has uploaded file with content "file to share with group" to "/fileToShareWithGroup.txt"
     And user "Alice" has uploaded file with content "file to share with public" to "/fileToShareWithPublic.txt"
-    And user "Alice" has shared folder "/folderToShareWithUser" with user "Brian"
-    And user "Alice" has shared folder "/folderToShareWithGroup" with group "grp1"
-    And user "Alice" has created a public link share with settings
-      | path        | /folderToShareWithPublic |
-      | permissions | read                     |
-      | password    | %public%                 |
-    And user "Alice" has shared file "/fileToShareWithUser.txt" with user "Brian"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | folderToShareWithUser |
+      | space           | Personal              |
+      | sharee          | Brian                 |
+      | shareType       | user                  |
+      | permissionsRole | Viewer                |
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | folderToShareWithGroup |
+      | space           | Personal               |
+      | sharee          | grp1                   |
+      | shareType       | group                  |
+      | permissionsRole | Viewer                 |
+    And user "Alice" has created the following resource link share:
+      | resource           | folderToShareWithPublic |
+      | space              | Personal                |
+      | permissionsRole    | view                    |
+      | password           | %public%                |
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | fileToShareWithUser.txt |
+      | space           | Personal                |
+      | sharee          | Brian                   |
+      | shareType       | user                    |
+      | permissionsRole | Viewer                  |
     And user "Brian" has accepted share "/fileToShareWithUser.txt" offered by user "Alice"
```

Comment:
this step should not be required, right?

Reply:
step `And user "Brian" has disabled auto-accepting` is used. so need to accept the share